partner/app/main.py

```diff
@@ -89,6 +107,25 @@ async def read_tilesp(
     results: int = Query(1, example=2),
 ):
     """Endpoint for requests from Contile."""
+    unknown_query_params = [
+        param for param in request.query_params if param not in ACCEPTED_QUERY_PARAMS
+    ]
+
+    if unknown_query_params:
+        logger.error(
+            "received unexpected query parameters from Contile: %s",
+            unknown_query_params,
+        )
+        return JSONResponse(
+            status_code=status.HTTP_400_BAD_REQUEST,
```

Comment:
Hi @ncloudioj! 👋🏻 I'd love to hear your thoughts on returning a `400 Bad Request` status to Contile when we receive an unexpected query parameter. Technically extra query parameters are allowed for many APIs, but for the sake of verifying that Contile is **not** sending more than what's required, we'd like to return an error. Do you think a `400` is good for that case or should be use a different client error status? Thanks!

Reply:
I think this is a great idea.

FWIW, adM's endpoint is quite tolerant on missing & unknown parameters, so it'll be nice to have this extra check with our unite & integration tests. Let's do it.

cc: @pjenvey @jrconlin